src/nxt_runtime.c

```diff
@@ -797,6 +802,60 @@ nxt_runtime_conf_init(nxt_task_t *task, nxt_runtime_t *rt)
     return NXT_OK;
 }
 
+static nxt_int_t
+nxt_runtime_creds(nxt_task_t *task,  nxt_runtime_t *rt)
+{
+    char          *user, *group_name;
+    struct group  *grp;
+    struct passwd *pwd;
+
+    pwd = getpwuid(geteuid());
+
+    if (pwd == NULL) {
+        user = group_name = NULL;
+    } else {
+        user = pwd->pw_name;
+        grp = getgrgid(pwd->pw_gid);
+
+        if (grp != NULL) {
+            group_name = grp->gr_name;
+        } else {
+            group_name = NULL;
+        }
+    }
+
+    if (rt->capabilities.setid) {
```

Comment:
It seems that if this condition is true, then all the code above is useless.

Reply:
done